data-service/build.gradle

```diff
@@ -71,7 +44,7 @@ dependencies {
             'org.apache.logging.log4j:log4j-slf4j-impl:2.8.+',
```

Comment:
can we move logging to dep management too? since there's no reason to use different versions across projects

Reply:
Agreed; I'll do that as part of the bigger logging cleanup to move to slf4j everywhere.